### DISCUSSION TITLE
Installing extensions for development environment

### DISCUSSION THREAD

**Discussion by magland** [jupyterlite/jupyterlite#1393] [Q&A]
I followed the instructions in [CONTRIBUTING.md](https://github.com/jupyterlite/jupyterlite/blob/main/CONTRIBUTING.md) to set up a development environment. When I run the server via

`jlpm serve`

I can connect via browser at port 5000. However, I am not able to see the pyodide kernel, etc. This is all I see

![image](https://github.com/jupyterlite/jupyterlite/assets/3679296/c37603b3-e6f4-42a6-b865-03d5e7e7ed59)

Do I need to install extensions somehow? It's not clear to me how to do that.

I am also interested in creating my own frontend extension.

Thanks!

**Comment by ianthomas23** [comment]
I experience the same on ARM macOS using the latest commit f1df061da.

Workflow:
1. `micromamba env create -f .binder/environment.yml`
2. `micromamba activate jupyterlite-dev`
3. `pip install jupyterlite-javascript-kernel`
4. `jlpm`
5. `jlpm build`
6. `jlpm serve`

The "Installing other kernels" section of `CONTRIBUTING.md` says to `pip install` them before running `jupyter lite build`. But there is no explicit `jupyter lite build` in the workflow, maybe it is hidden behind one of the `jlpm` or `doit` commands?
